src/Microsoft.AspNetCore.Localization/Microsoft.AspNetCore.Localization.csproj

```diff
@@ -17,6 +17,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Http.Extensions" Version="$(AspNetCoreVersion)" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="$(AspNetCoreVersion)" />
```

Comment:
Is this for the `NullLogger` instance? I'd rather we avoid adding package references and just check for null `ILogger` instances in the code.